### PR TITLE
MLPA-115 - Create package update PRs for all ecosystems

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Keep packages up to date

Fixes MLPAB-115

## Approach

- add a Dependabot updates configuration

## Learning

- https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

## Checklist

* [x] I have performed a self-review of my own code
* ~I have added relevant logging with appropriate levels to my code~
* ~I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* ~I have added tests to prove my work~
* ~I have added welsh translation tags and updated translation files~
* ~I have run an accessibility tool on any pages I have made changes to and fixed any issues found~
* ~The product team have tested these changes~

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      separator: "-"
+    directory: '/'
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'composer'
+    pull-request-branch-name:
+      separator: "-"
+    directory: '/'
+    schedule:
+      interval: 'daily'
```
Comment (acsauk) [inline .github/dependabot.yml:26]: This looks like its PHP related - hopefully not going to need that in this repo 🙏

Comment (andrewpearce-digital) [reply .github/dependabot.yml:26]: yeah, I've left all ecosystems in place on purpose

Comment (andrewpearce-digital) [reply .github/dependabot.yml]: removed :D

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**